features/RuntimeConfiguration.feature.md

```diff
@@ -281,18 +287,16 @@ And there is this device shadow data for `${fingerprint_deviceId}` in nRF Cloud
 
 When I connect to the websocket using fingerprint `${fingerprint}`
 
-<!-- @retry:tries=5,initialDelay=5000,delayFactor=1.5 -->
-
 Soon the duration between 2 consecutive device shadow requests for
 `${fingerprint_deviceId}` should be `5` seconds
 
 ## Verify if changing fetching interval to 10 seconds, device shadow requests should be fired 1 time in 10 seconds
 
 Given device shadow fetching config for model `PCA20035+solar` is `10`
 
-When I connect to the websocket using fingerprint `${fingerprint}`
+When I wait for `5` seconds
```

Comment:
This is nothing the user should do, the 5 second delay should be handled in the following connect step. It's an implementation detail of the test system and should not be made part of the public API.

Reply:
But if we put the logic in web socket connection, it will apply for all even we need it for a specific case.

Reply:
Yes, it's the DB that is lacking behind so this needs an internal retry for the connection db.

Reply:
And this check should be in the promise of the step that **provides** the data in the db. So this step waits until it is available and only then do the next steps execute. If the step is later called, the promise is already fulfilled and it will continue immediately.